modules/ROOT/pages/index.adoc

```diff
@@ -2,14 +2,40 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:page-aliases: overview-4.adoc
 
-APIkit is a tool for building Mule REST or SOAP APIs. You develop APIs based on one of the following modeling languages:
+APIkit is a toolkit that simplifies the development of API-led applications for Mule developers. It scaffolds applications from a given specification, validates inputs parameter validation, and creates xref:munit::munit-scaffold-test-task.adoc[MUnit test suites]. APIkit has a module for each supported web-service type:
```

Comment:
According to our guidelines for writing overview, the topic title should be "APIkit Overview" instead of "About APIkit."

Reply:
OK! Thanks!